services-js/commissions-app/src/server/commissions-app.ts

```diff
@@ -163,18 +164,23 @@ export async function makeServer(port, rollbar: Rollbar) {
   server.route({
     method: 'POST',
     path: '/commissions/submit',
+    options: {
+      payload: {
+        // Two 5mb files in bytes, plus another 500k for other form information.
```

Comment:
If we’re changing the max accepted value, we’ll also need to change the helper text, as well as the `sizeLimit` property on the two `FileInput`s.

https://github.com/CityOfBoston/digital/blob/18f5d780b54e9f6613f76c8ca05cd2acf6727519/services-js/commissions-app/src/client/ApplicationForm.tsx#L152
https://github.com/CityOfBoston/digital/blob/18f5d780b54e9f6613f76c8ca05cd2acf6727519/services-js/commissions-app/src/client/ApplicationForm.tsx#L158

Reply:
Good catch. Updated in those two places.